Assets/MixedRealityToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -153,6 +174,12 @@ public virtual void OnInputClicked(InputClickedEventData eventData)
             eventData.Use();
         }
 
+        private void RaisePlacingEvent(ExecuteEvents.EventFunction<ITapToPlaceHandler> handler, GameObject selectedObject)
```

Comment:
This method and it's delegate counterpart should me moved into the input manager class.

Reply:
@StephenHodgson in fact that was my initial approach but then I noticed that all current events implement/need the corresponding IInputSource and I was not sure what to do for the case of the new IPlacementHandler. Any thoughts on that?

Reply:
No need to add a new source.  The input source is just as it says, the root source of the input, which in this case would be from the Tap event (passing in the event data's input source should be enough to satisfy this condition.) Thoughts?

Reply:
Check out the new iteration and tell me what you think.